sound.js

```diff
@@ -279,8 +279,6 @@ Sound.prototype.setCurrentTime = function(value) {
 Sound.prototype.setSpeakerphoneOn = function(value) {
   if (IsAndroid) {
     RNSound.setSpeakerphoneOn(this._key, value);
-  } else if (!IsAndroid && !IsWindows) {
-    RNSound.setSpeakerphoneOn(value);
```

Comment:
`setSpeakerphoneOn` for ios is only supported in our repo. Original `react-native-sound` has no `setSpeakerphoneOn` for ios. Thus, we don't worry backward compatibility.

Reply:
I think we need to choose one of the following:

1. Support this method for iOS and Android. **<-- I personally prefer this! See the reason below**
2. Support this method for iOS and Android and use a different name for a static method (not sure which name is appropriate).
3. Provide `console.wran('Use static method Sound.setSpeakerphoneOn() instead for iOS!')` here for developers that the call will be ignored.

I think our use case in https://github.com/hayanmind/mimiking-rn/pull/1836 is a little special in the standpoint of this `react-native-sound`. So, I believe supporting instance method makes more sense.

Reply:
Okay, then how about the name `Sound.setSpeakerphoneOnIos()` for static method?